source/agora/consensus/EnrollmentManager.d

```diff
@@ -146,10 +145,10 @@ public class EnrollmentManager
 
     ***************************************************************************/
 
-    public bool add (ulong block_height, scope UTXOFinder finder,
-        const ref Enrollment enroll) @safe nothrow
+    public bool add (scope UTXOFinder finder, const ref Enrollment enroll)
+        @safe nothrow
     {
-        return this.validator_set.add(block_height, finder, enroll);
+        return this.validator_set.add(0, finder, enroll);
```

Comment:
The `block_height` parameter should also be removed from `ValidatorSet.add`, it's unused now.

Reply:
Discussed IRL, this will be addressed here: https://github.com/bpfkorea/agora/pull/748/files#diff-a1aa02c9e0b910c82366388fb0900864R154-R158